src/pint/scripts/pintk.py

```diff
@@ -77,6 +79,34 @@ def initUI(self):
         self.helpMenu.add_command(label="Plk Help", command=lambda: print(helpstring))
         self.menuBar.add_cascade(label="Help", menu=self.helpMenu)
 
+        self.logMenu = tk.Menu(self.menuBar)
```

Comment:
I'm a little confused - does this provide a list of individually selectable checkboxes, or a list of options for the user to choose between? If I understand the logging mechanism, setting its log level to "INFO" enables info messages and also all the more important categories, so these categories aren't really selectable independently. I think tkinter's Combobox widget allows selecting between a finite list of options in a familiar way.

Reply:
This is a menu of checkboxes, and how I have it set up is it essentially acts like a Combobox widget. Clicking the Log Level menu drops down a list of options with the selected option having a checkmark by it. Selecting another option unselects the current option and places the checkmark by that option. This is the minimum log level outputted - maybe changing the menu name to "Minimum Log Level" would be more understandable?

Reply:
Why not use a Combobox?

Reply:
I can change it to a combobox, I was just maintaining uniform visual, but shouldn't be a problem to switch. I can do that later tonight.

Reply:
I replaced the menu, I couldn't place it in the menu bar at the top of the window so I placed it at the bottom. It hangs off the bottom of the window when expanded, if that is bothersome I am open to suggestions for other placements.

Reply:
The normal behaviour of combo boxes is to pop up a list to choose from, which indeed can hang outside the parent window. Would tkinter's OptionMenu suit your case better?

Reply:
I can definitely use that, I don't really have a preference though. I suppose it comes down to if you want the drop-down to be at the top of the window or the bottom - do you or @paulray have a preference?

Reply:
My only objection was to reimplementing Combobox/OptionMenu by hand when code already existed.